extensions/ql-vscode/src/eval-log-tree-builder.ts

```diff
@@ -0,0 +1,66 @@
+import { ChildEvalLogTreeItem, EvalLogTreeItem } from './eval-log-viewer';
+import { EvalLogData as EvalLogData } from './pure/log-summary-parser';
+
+export default class EvalLogTreeBuilder {
```

Comment:
Or something to this effect.
```suggestion
/** Builds the tree data for the evaluator log viewer for a single query run. */
export default class EvalLogTreeBuilder {
```

Reply:
Thanks! I was just thinking that I didn't add many class- or function-level comments this time around. I'll do a final check for other places where comments might be helpful before merging.